### PR TITLE
Bengali translation: to link incomplete Bengali translation

### PR DESCRIPTION
**Pull request by raselmandol**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ concurrency:
 env:
   CARGO_TERM_COLOR: always
   # Update the language picker in index.hbs to link new languages.
-  LANGUAGES: da de fr it ja ko pt-BR
+  LANGUAGES: da de fr it ja ko pt-BR bn
```
Comment (mirwasi) [inline .github/workflows/publish.yml:22]:
```suggestion
  LANGUAGES: bn da de fr it ja ko pt-BR
```

Comment (mirwasi) [reply .github/workflows/publish.yml:22]: Keep them sorted.

File: src/running-the-course/translations.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ recently updated translations:
 * [French][fr] by [@KookaS] and [@vcaen].
 * [German][de] by [@Throvn] and [@ronaldfw].
 * [Japanese][ja] by [@CoinEZ-JPN] and [@momotaro1105].
+* [Bengali][bn] by [@raselmandol].
```
Comment (mirwasi) [inline src/running-the-course/translations.md:19]:
```suggestion
* [Bengali][bn] by [@raselmandol].
* [French][fr] by [@KookaS] and [@vcaen].
* [German][de] by [@Throvn] and [@ronaldfw].
* [Japanese][ja] by [@CoinEZ-JPN] and [@momotaro1105].
```

File: src/running-the-course/translations.md [hunk 3]
```diff
@@ -25,6 +26,7 @@ get going. Translations are coordinated on the [issue tracker].
 [ja]: https://google.github.io/comprehensive-rust/ja/
 [ko]: https://google.github.io/comprehensive-rust/ko/
 [pt-BR]: https://google.github.io/comprehensive-rust/pt-BR/
+[bn]: https://google.github.io/comprehensive-rust/bn/
```
Comment (mirwasi) [inline src/running-the-course/translations.md:29]:
```suggestion
```

Comment (mirwasi) [inline src/running-the-course/translations.md:29]: Can't make a suggested change here, but again, make them sorted (bn should come first)

**Comment by raselmandol** [comment]
@mirwasi ,I have fixed the problems(sorted)

**Review by mirwasi — APPROVED** [review]

**Comment by mgeisler** [comment]
Looks good, thanks for keeping things tidy! :smile:

**Review by HAMMERABA — APPROVED** [review]

**Review by HAMMERABA — APPROVED** [review]

**Review by HAMMERABA — APPROVED** [review]

**Review by HAMMERABA — APPROVED** [review]

**Pull request merged**